### PR TITLE
Add operator manifest

### PR DESCRIPTION
**Pull request by imcsk8**
This manifest is needed in order to run the
Cluster Ingress Operator as a container inside
OpenShift.

Also added running information to the documentation.

### CODE REVIEW

**Comment by imcsk8** [comment]
PTAL @openshift/sig-network-edge @ironcladlou @pravisankar @Miciah

File: deploy/cluster-ingress-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      containers:
+        - name: cluster-ingress-operator
+          image: openshift/cluster-ingress-operator
+          ports:
+          - containerPort: 60000
+            name: metrics
+          command:
+          - cluster-ingress-operator
+          imagePullPolicy: Always
+          env:
```
Comment (ironcladlou) [inline deploy/cluster-ingress-operator.yaml]: Isn't `KUBERNETES_CONFIG` also needed?

Comment (imcsk8) [reply deploy/cluster-ingress-operator.yaml]: I haven't needed it in my tests. I just run it inside a openshift environment as the system user.
```
oc create -f deploy/cluster-ingress-operator.yaml
```

**Pull request merged**